src/components/ProfilePage.js

```diff
@@ -127,125 +101,144 @@ export default function ProfilePage() {
+                            <Form.Label>Interest </Form.Label>
+                            <Form.Control as="textarea" rows={3} defaultValue = {interests} readOnly/>
+                            </Form.Group>
+                            </Form>
+                        </Col>
+                        <Col>
+                            <Form>
+                            <Form.Group className="mb-3" controlId="exampleForm.ControlTextarea1">
+                            <Form.Label>Contact Information</Form.Label>
+                            <Form.Control as="textarea" rows={3} defaultValue = {links} readOnly/>
+                            </Form.Group>
+                            </Form>
+                        </Col>
+                    </Row>
+                    </div>
+                )
+            }
             {
                 canEdit && (
-                    <div className = "edit-save-buttons">
-                    <Form.Group className="mb-3" controlId="submitform">
-                        <Button variant="outline-info" type="submit" onClick={editProfile}>
-                            Update
-                        </Button> 
-                    </Form.Group>
-
-            </div>
+                    <div>
+                    <Row>
+                        <Col>           
+                            <Form>
+                            <Form.Group>
+                            <Form.Label>First Name</Form.Label>
+                            <Form.Control type="text" defaultValue = {firstName} onChange = {(e) => (setFirstName(e.target.value))}/>
+                            <Form.Label>Last Name</Form.Label>
+                            <Form.Control type="text" defaultValue = {lastName} onChange = {(e) => (setLastName(e.target.value))}/>
+                            </Form.Group>
+                            <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
+                            <Form.Label>Email</Form.Label>
+                            <Form.Control type="text" defaultValue = {email} onChange = {(e) => (setEmail(e.target.value))}/>
+                            </Form.Group>
+                            <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
+                            <Form.Label>Occupation</Form.Label>
+                            <Form.Control type="text" defaultValue = {occupation} onChange = {(e) => (setOccupation(e.target.value))}/>
+                            </Form.Group>
+                            </Form>       
+                        </Col>
+
+                        <Col>
+                            <Form>
+                            <Form.Group className="mb-3" controlId="exampleForm.ControlTextarea1">
+                            <Form.Label>About Me</Form.Label>
+                            <Form.Control as="textarea" rows={3} placeholder="Tell me about yourself." defaultValue = {bio} onChange = {(e) => (setBio(e.target.value))}/>
```

Comment:
Instead of placeholder text, we can have gray text under the section heading/label to avoid whitespace. It will also look cleaner if we use the latter.